server/src/util/fs.ts

```diff
@@ -11,3 +12,10 @@ export function getStats(path: string): Promise<Fs.Stats> {
     })
   })
 }
+
+export function untildify(pathWithTilde: string): string {
+  const homeDirectory = Os.homedir()
```

Comment:
I would a comment with a reference to where this comes from. I assume https://github.com/sindresorhus/untildify/blob/master/index.js#L11